lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -181,7 +181,7 @@ class << self
 
           sig { override.returns(T::Enumerable[Module]) }
           def gather_constants
-            ActiveRecord::Base.descendants.reject(&:abstract_class?)
+            ActiveRecord::Base.descendants
```

Comment:
While we are at it, can we please change this to be:
```suggestion
            descendants_of(ActiveRecord::Base)
```
as well? Thanks!